pgbackrest_auto

```diff
@@ -647,16 +661,22 @@ if [[ "${CHECKDB_MODE}" != "No" ]]; then
     if [[ "${CHECKDB}" = "yes" || "${CHECKSUMS}" = "yes" ]]; then
     info "[STEP $((STEP++))]: Verify data checksums"
     pg_checksums
+    else
+    sed -i 's/PG_checksums_validation=0/PG_checksums_validation=1/g' ${zbx_status_file}
```

Comment:
this seems redundant, since the pg_checksums function also writes statuses

Reply:
before new restore all values resets in zabbix status file (line 602)